### DISCUSSION TITLE
[Quiz] what is the name of the mascot for penguinmod?

### DISCUSSION THREAD

**Discussion by polygoners1** [PenguinMod/PenguinMod-Home#118] [Polls]
The mascot is the default sprite when you make a new project in penguinmod.
The deadline is august 1st 2024.

**Comment by Probioticz** [comment]
i wish it was "Mod"

**Comment by jwklong** [comment]
> i wish it was "Mod"

are we deadass

**Reply by Probioticz** [reply]
wdym deadass?
